vyper/codegen/ir_node.py

```diff
@@ -13,6 +13,8 @@
 from vyper.semantics.types import VyperType
 from vyper.utils import VALID_IR_MACROS, ceil32
 
+PRECOMPILE_RANGE = (1, 10)
```

Comment:
might not be true for some L2s

Reply:
i thought about this (see also eip-7587) but i think it's ok.
1. we can't predict every precompile that will be in use by an L2
2. precompiles could have arbitrary effects
3. our main goal here is to protect against precompiles which could be generated in the compiler. user-generated calls to precompiles are fair game (we can consider them "unsafe")